app/src/main/java/swati4star/createpdf/util/FileUriUtils.java

```diff
@@ -177,11 +177,19 @@ static String getImageRealPath(ContentResolver contentResolver, Uri uri, String
      * @param uri - input uri
      * @return - path
      */
-    public static String getFilePath(Uri uri) {
+    public String getFilePath(Uri uri) {
         String path = uri.getPath();
         if (path == null)
             return null;
         path =  path.replace("/document/raw:", "");
         return path;
     }
+
+    private static class SingletonHolder {
+        static final FileUriUtils INSTANCE = new FileUriUtils();
+    }
+
+    public static FileUriUtils getInstance() {
```

Comment:
make this class's contructor private?

Reply:
Will make it and push the changes again.